mail/mail.go

```diff
@@ -2,234 +2,45 @@ package mail
 
 import (
 	"bytes"
-	"crypto/rand"
 	"encoding/hex"
-	"fmt"
 	"io/ioutil"
-	"net/http"
-	"net/mail"
 	"text/template"
-	"time"
 
 	"golang.org/x/net/context"
-	"google.golang.org/appengine"
-	"google.golang.org/appengine/datastore"
-	"google.golang.org/appengine/log"
 	appmail "google.golang.org/appengine/mail"
 )
 
-type Subscription struct {
-	Address          string
-	EntryTime        time.Time
-	Token            []byte
-	VerificationTime time.Time
+func SendMail(c context.Context, to []string, subject, body string) error {
+	return appmail.Send(c, &appmail.Message{
+		Sender:  "Lorenz Leutgeb <lorenz.leutgeb@cod.uno>",
+		To:      to,
+		Subject: subject,
+		Body:    body,
+	})
 }
 
-var verify = template.New("verify")
-
-func init() {
-	// Add function map to template engine
+//PrepareMailTemplate parses the mail template defined and fills it with data
+func PrepareMailTemplate(templatePath string, data interface{}) (string, error) {
```

Comment:
This code was in `init` on purpose to be executed only once. Now it is called by `.../subscription.Subscription.RequestConfirmation` which in turn is called by `.../subscription.subscribe` which is a handler, so we will collect a lot of garbage and not allocate memory globally.